src/Microsoft.Extensions.Configuration/IncludeConfigurationProvider.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Extensions.Configuration
+{
+    public class IncludeConfigurationProvider : ConfigurationProvider
```

Comment:
> I think its clearer for the name to exactly match the extension method that adds it, since these classes aren't used directly

Users still get exposed to this class classes even if less frequently than the extension method. We should pick a name that makes some sense.